battleship-server/src/main/java/com.epam.solid.nie/server/ShipSocketServer.java

```diff
@@ -6,16 +6,25 @@
 import java.net.Socket;
 
 class ShipSocketServer implements ShipServer {
-    private final int portNumber = 8080;
+
+    private final String ip;
+
+    ShipSocketServer(String ip) {
+        this.ip = ip;
+    }
 
     @Override
     public void run() {
-        try (ServerSocket serverSocket = new ServerSocket(portNumber, 0, InetAddress.getLoopbackAddress())) {
+        int portNumber = 8080;
```

Comment:
private final int variable was good idea

Reply:
so should it be a field in your opinion?